README.md

```diff
@@ -34,6 +34,13 @@ The direction is adjacent to [Adept.ai](https://adept.ai/), with some key differ
 
 ## Install
 
+Required dependencies:
+- Python 3.10
+- Git
+- Tesseract (for OCR)
+
+Follow the setup instructions in setup.md for your operating system.
```

Comment:
@KrishPatel13 is there a setup.md?

Reply:
I will be adding soon.

Reply:
Addressed in: [8dc42c1](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/8dc42c1feefd4e39aab70f1ff453a15872832dfb)